src/_posts/2021-10-04-september-steering-committee-recap.md

```diff
@@ -54,7 +54,7 @@ While the Steering Committee hopes for clarity from Code for America around its
 
 ## Discussion: Project requirements
 
-The initial pilot of our [Project Team Scan](https://docs.google.com/presentation/d/1lADVpWHaqQKSd05mlCtsYvk8rd2Jlg-iMhSwxQGvBa4/edit?usp=sharing) was intended to gather critical project details to enable Steering Committee oversight and public access to projects, and to gauge the health of project teams. The results revealed that while most projects make their code and documentation publicly accessible per the open source guidelines described in OpenOakland’s bylaws, Councilmatic currently does not have an up-to-date Github repository or recent documentation available to the public. Councilmatic has also not sent a project rep to Steering Committee meetings since January 2021, despite the bylaws [mandating that project teams send a voting representative](https://docs.google.com/document/d/1QR-fr1WnmXkZoVNmWnZ9drzfmaZoPkodEOx-PkExt94/edit#heading=h.djbozkxsdpw8) in an effort to ”give back and actively shape the organization to meet its needs.” September’s discussion focused on how best to address this in a way that enables Councilmatic to become a productive, contributing member of the brigade family of projects.
+The initial pilot of our [Project Team Scan](https://docs.google.com/presentation/d/1lADVpWHaqQKSd05mlCtsYvk8rd2Jlg-iMhSwxQGvBa4/edit?usp=sharing) was intended to gather critical project details to enable Steering Committee oversight and public access to projects, and to gauge the health of project teams. The results revealed that while most projects make their code and documentation publicly accessible per the open source guidelines described in OpenOakland’s bylaws, Councilmatic currently does not have an up-to-date Github repository or recent documentation available to the public. Councilmatic has also not sent a project rep to Steering Committee meetings since January 2021, despite the [bylaws](/bylaws) mandating that project teams send a voting representative in an effort to ”give back and actively shape the organization to meet its needs.” September’s discussion focused on how best to address this in a way that enables Councilmatic to become a productive, contributing member of the brigade family of projects.
```

Comment:
Isn't Councilmatic in compliance now? Is this still relevant?

Reply:
This was simply updating the link in an old blog post to the new ByLaws page.